index.js

```diff
@@ -33,8 +38,10 @@ Plugins.prototype = Emitter({
       next = opts;
       opts = null;
     }
-    argv = this.minimist(argv, opts);
-    this.plugins.run(argv, function (err, args) {
```

Comment:
I don't think this will work the way you expect. that's why I didn't do this before. the problem is that the options on the instance (set in the constructor) may not be the same at the time that the plugin is run. IMHO it's more predictable and reliable to just require plugins to set their own options, and or pass the options in outside of the instance. like you would do with gulp plugins. @doowb?